corehq/apps/export/models/new.py

```diff
@@ -156,6 +157,7 @@ class ExportItem(DocumentSchema):
     tag = StringProperty()
     last_occurrences = DictProperty()
     transform = StringProperty(choices=list(TRANSFORM_FUNCTIONS))
+    datatype = StringProperty()  # this is not used by exports, but other things that use this schema (e.g. app-based UCRs)
```

Comment:
If this is becoming more than just an "Export Schema", and a more canonical way of getting data about an app, would we want to consider moving it out of the exports code and making it its own module?

Reply:
Yes, I'd definitely be in favor of this, though unclear when to do it and to what degree it will be worth the ROI (I haven't really explored how hard that would be yet).

Reply:
I also like this idea - feels very in-line with where we're heading around having real data schemas for projects.